### PR TITLE
test: Add k8s test to copy file from pod to host

### LINKED ISSUE #2096: Add k8s test to copy file from pod to host

### PR DESCRIPTION
**Pull request by GabyCT**
This will test that we are able to copy a file from a k8s pod to a host.

Fixes #2096

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by GabyCT** [comment]
Related https://github.com/kata-containers/runtime/issues/2061

File: integration/kubernetes/k8s-copy-file.bats [hunk 1]
```diff
@@ -33,6 +33,24 @@ setup() {
...
+	kubectl create -f "${pod_config_dir}/pod-env.yaml"
+
+	# Check pod creation
+	kubectl wait --for=condition=Ready pod "$pod_name"
+
+	# Create a file in the pod
+	kubectl exec $pod_name -- sh -c "cd /tmp && echo $content > $file_name"
+
+	# Copy file from pod to host
+	kubectl cp $pod_name:tmp/$file_name $file_name
```
Comment (grahamwhaley) [inline integration/kubernetes/k8s-copy-file.bats]: tiny nit: should that be `/`tmp/$file_name ?

Comment (chavafg) [reply integration/kubernetes/k8s-copy-file.bats]: please also quote variables

Comment (GabyCT) [reply integration/kubernetes/k8s-copy-file.bats]: @chavafg changes applied

**Pull request merged**